modules/microshift-disconnected-host-procedure.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-disconnected-network-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disconnected-host-network-config_{context}"]
+= Configuring the networking settings for fully disconnected hosts
+
+To configure the networking settings for running {microshift-short} on a fully disconnected host, you must prepare the host, update the networking configuration, then restart to apply the new settings. All commands are executed from the host CLI.
+
+.Prerequisites
+* RHEL 9 or newer.
+* {microshift-short} 4.14 or newer.
+* Access to the host CLI.
+* A valid IP address chosen to avoid both internal and potential future external IP conflicts when running {microshift-short}.
+* {microshift-short} networking settings are set to defaults.
+
+.Procedure
+
+. Add a fake IP address to the loopback interface by running the following command:
++
+[source,terminal]
+----
+$ IP="10.44.0.1" <1>
+$ sudo nmcli con add type loopback con-name stable-microshift ifname lo ip4 ${IP}/32
+----
+<1> The fake IP address used in this example is “10.44.0.1”.
++
+[NOTE]
+====
+Any valid IP works, as long as it avoids both internal {microshift-short} and potential future external IP conflicts. This can be any subnet that does not collide with the {microshift-short} node subnet or is be accessed by other services on the device.
+====
+
+. Configure the DNS interface to use the local name server by setting modifying the settings to ignore automatic DNS and reset it to the local name server:
++
+.. Bypass the automatic DNS by running the following command:
++
+[source,terminal]
+----
+$ sudo nmcli conn modify stable-microshift ipv4.ignore-auto-dns yes
+----
++
+.. Point the DNS interface to use the local name server:
++
+[source,terminal]
+----
+$ sudo nmcli conn modify stable-microshift ipv4.dns "10.44.1.1"
+----
+
+. Get the hostname of the device by running the following command:
++
+[source,terminal]
+----
+$ NAME="$(hostnamectl hostname)"
+----
+
+. Add an entry for the hostname of the node in the `/etc/hosts` file by running the following command:
++
+[source,terminal]
+----
+$ echo "$IP $NAME" | sudo tee -a /etc/hosts >/dev/null
+----
+
+. Update the {microshift-short} configuration file by adding the following YAML snippet to `/etc/microshift/config.yaml`:
++
+[source,terminal]
+----
+sudo cat <<EOF >> /etc/microshift/config.yaml
```

Comment:
This is going to have the permission denied issue.

I am suggesting

`sudo tee /etc/microshift/config.yaml > /dev/null <<EOF 
node:
  hostnameOverride: $(echo $NAME)
  nodeIP: $(echo $IP)
EOF`